scripts/protocgen.sh

```diff
@@ -12,6 +12,17 @@ Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
   $(find "${dir}" -maxdepth 1 -name '*.proto')
 done
 
+# TODO update the command to include ./proto/ibc too and add the following commands in above loop
```

Comment:
It's okay to put in this PR, but this will be useful only in the next PR with the actual gateway server.

So I would recommend:
- fix this TODO, and leave it in this PR
- or remove this piece of code, and add it in the next PR